nmrfx-core/src/main/java/org/nmrfx/peaks/types/PeakListTypeDim.java

```diff
@@ -1,9 +1,12 @@
 package org.nmrfx.peaks.types;
 
+import org.nmrfx.annotations.YamlEntity;
+
+@YamlEntity("peakpat")
```

Comment:
Intellij was falsely detecting this class as unused because it couldn't find a constructor usage. 
It's actually instanciated by reflection by snakeyaml.